src/card-stack/card-stack.html

```diff
@@ -37,11 +66,18 @@
         <game-card
           draggable="true"
           on-drag="_dragStart"
-          on-drop="_drop"
           position$="[[index]]"
           rank="[[item.rank]]"
           symbol="[[_symbolForSuit(item.suit)]]"></game-card>
       </template>
+      <template is="dom-if" if="[[_isFoundation()]]">
```

Comment:
This likely doesn't work how you think it does. Without passing a value in your binding this gets stuck at whatever the return value was at the time it was bound. You mean `if="[[foundation]]"`, likely.